bika/lims/browser/multifile.py

```diff
@@ -1,82 +0,0 @@
-# -*- coding: utf-8 -*-
-#
-# This file is part of SENAITE.CORE
-#
-# Copyright 2018 by it's authors.
-# Some rights reserved. See LICENSE.rst, CONTRIBUTORS.rst.
-
-from zope.interface import implements
-from bika.lims.browser.bika_listing import BikaListingView
-from plone.app.content.browser.interfaces import IFolderContentsView
-from plone.app.layout.globals.interfaces import IViewView
-from bika.lims import bikaMessageFactory as _
-
-
-class MultifileView(BikaListingView):
```

Comment:
Be careful!, seems that this `MultifileView` is being used in health, here: https://github.com/senaite/senaite.health/blob/master/bika/health/browser/patient/files.py

Reply:
No worries, will create a PR myself against health